addons/website/static/tests/tours/html_editor.js

```diff
@@ -6,6 +6,42 @@ import wTourUtils from "@website/js/tours/tour_utils";
 const adminCssModif = '#wrap {display: none;}';
 const demoCssModif = '// demo_edition';
 
+wTourUtils.registerWebsitePreviewTour('html_editor_language', {
+    url: '/test_page',
+    test: true,
+}, () => [
+    {
+        content: "open site menu",
+        extra_trigger: "iframe body:not(.editor_enable)",
```

Comment:
Nitpickg but always nice sticking to it to save an indent level:
```suggestion
}, () => [{
    content: "open site menu",
    extra_trigger: "iframe body:not(.editor_enable)",
```